Admin/Extension/RouteReferrersExtension.php

```diff
@@ -1,44 +0,0 @@
-<?php
-
-/*
- * This file is part of the Symfony CMF package.
- *
- * (c) 2011-2015 Symfony CMF
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- */
-
-namespace Symfony\Cmf\Bundle\RoutingBundle\Admin\Extension;
-
-use Sonata\AdminBundle\Admin\AdminExtension;
-use Sonata\AdminBundle\Form\FormMapper;
-use Sonata\CoreBundle\Form\Type\CollectionType;
-
-/**
- * Admin extension to add routes tab to content implementing the
- * RouteReferrersInterface.
- *
- * @author David Buchmann <mail@davidbu.ch>
- */
-class RouteReferrersExtension extends AdminExtension
-{
-    public function configureFormFields(FormMapper $formMapper)
-    {
-        $formMapper
-            ->with('form.group_routes', array(
-                'translation_domain' => 'CmfRoutingBundle',
-            ))
-            ->add(
-                'routes',
-                CollectionType::class,
-                array(),
-                array(
-                    'edit' => 'inline',
-                    'inline' => 'table',
-                )
```

Comment:
@dbu @WouterJ can you give me a hand on replacing that by a custom form type. I am ablte to create a form type with fields like we have in the admin, but how should i add them here for several fields instead of a single field as i know it?